src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -114,7 +117,9 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
   }
 
   protected firstUpdated(): void {
-    this._fetchPersonNames();
+    if (this.hass) {
```

Comment:
Remove this check and change `hass?` to `hass!`

Reply:
This is fixed.

Thank you for your time and valuable comments! All the details in these PRs together go a long way.